src/main/java/com/hm/achievement/AdvancedAchievements.java

```diff
@@ -678,6 +680,11 @@ private void registerListeners() {
 
 		playerAdvancedAchievementListener = new PlayerAdvancedAchievementListener(this);
 		pm.registerEvents(playerAdvancedAchievementListener, this);
+		
+		if (Bukkit.getPluginManager().isPluginEnabled("PlaceholderAPI"))
```

Comment:
Could this block of code please be placed next to the `BungeeTabListPlusBukkitAPI` bit further up in the class? See my comment below about it not really being a listener.